COMPATIBILITY.md

```diff
@@ -30,26 +32,30 @@ The key elements are : 
+v1.4.x and v2.5.0 Java Libraries are supported running in Java 11 with the x86_64 architecture. Later Java 11 versions are supported but are not tested.
 
 v1.4.x Java Libraries are supported running in Java 8 with the  x86_64 architecture. Later Java 8 versions are supported but are not tested.
 
 Architecture Support: all docker images, JVMs, tools are tested under x86_64 ONLY
 
+
+
 ### Default Peer Runtime selection
 
-When using Fabric 2.1.0, the default docker image that is used to run the Java chaincode is *openjdk11:jdk-11.04_11-alpine*
+When using Fabric 2.5.0, the default docker image that is used to run the Java chaincode is *eclipse-temurin:11.0.16.1_1-jdk*
+
+With the default docker image used by Fabric 2.5.0. should the packaged Java code contain a Maven or Gradle build script, it will be built using Gradle 7.0, or Maven 3.8.1 (if both Gradle and Maven files are present Gradle is used.  Gradle build files can be groovy, or kotlin.  If the Gradle wrapper is present, this will used in preference to the installed version of Gradle; note that the version of Gradle in the docker image is a copy of the Gradle Wrapper, not the full Gradle install. Therefore if there is limited Internet acccess in your production environment, do not use Gradle). Alternatively it is recommended to package prebuilt jar files, including the contract and all dependencies, in which case no build or Internet access is required when installing Java chaincode.
 
-With the default docker image used by Fabric 2.1.0. should the packaged Java code contain a Maven or Gradle build script, it will be built using Gradle 5.6.2, or Maven 3.6.2 (if both Gradle and Maven files are present Gradle is used.  Gradle build files can be groovy, or kotlin.  If the Gradle wrapper is present, this will used in preference to the installed version of Gradle; note that the version of Gradle in the docker image is a copy of the Gradle Wrapper, not the full Gradle install. Therefore if there is limited Internet acccess in your production environment, do not use Gradle). Alternatively it is recommended to package prebuilt jar files, including the contract and all dependencies, in which case no build or Internet access is required when installing Java chaincode.
+Please check the [Dockerfile](./fabric-chaincode-docker/Dockerfile) that is used for the environment to see exactly how these versions are installed. 
 
 ### Supported Runtime communication with the Peer
  
-Subject to a suitable runtime environment, the 1.4.4 and 2.1.0 Java Libraries can used to communicate with a Fabric 2.1.0 or 1.4.4 Peer - with the level of functionality that is implied by the Fabric version in use. 
\ No newline at end of file
+Subject to a suitable runtime environment, the 1.4.4 and 2.5.0 Java Libraries can used to communicate with Fabric 2.5 and previous LTS versions (2.2.0/1.5.4) - with the level of functionality that is implied by the Fabric version in use and channel capabilities. 
```

Comment:
I'm confused... 1.4.4 and 2.5.0 Fabric versions not Java Libraries right?
And what is 1.5.4?
And what do channel capabilities have to do with this?

Reply:
yes Fabric versions.... 1.5.4 was a typo.

Even if the peer is 2.5 etc.. if you've configured channel capability to v2, for example you can't use purgepd.